shell/browser/native_window_views.cc

```diff
@@ -281,7 +281,13 @@ NativeWindowViews::NativeWindowViews(const gin_helper::Dictionary& options,
       new ElectronDesktopWindowTreeHostLinux(this, native_widget);
 #endif
 
+  // We want to set the can_resize param before initializing
+  // the widget, but we want to do it without emitting any events.
```

Comment:
Please explain why we want `can_resize` to be true before initializing the widget.

Reply:
Actually, I never tried setting it to true directly. I will have to confirm whether that could actually temporarily resolve the non-resizable scenario as well.

Reply:
If we want the issue to be resolved for non-resizable windows as well, then it turns out we have to move the `SetCanResize` call to after the `SetWindowLong` call in this file. Worse would be to patch Chromium to add in `WS_THICKFRAME` styles. I clarified the comments.